mysql-test/suite/rocksdb/t/deadlock_tracking.test

```diff
@@ -3,6 +3,8 @@ set @prior_deadlock_detect = @@rocksdb_deadlock_detect;
 set @prior_max_latest_deadlocks = @@rocksdb_max_latest_deadlocks; 
 set global rocksdb_deadlock_detect = on; 
 set global rocksdb_lock_wait_timeout = 10000;
+set global rocksdb_max_latest_deadlocks = 0;
```

Comment:
Can you add a comment about why this is done and also update the variable so that it's recognized what setting it to 0 does.